### PR TITLE
Fonts mixin

### LINKED ISSUE #18: Font face mixin

### PR DESCRIPTION
**Pull request by austinstarin**
Close #18

### CODE REVIEW

File: src/sass/tools/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@mixin generate-font ($name, $url, $file, $style: normal, $weight: normal) {
+  @font-face {
+    font-family: $name;
+    font-weight: $weight;
+    font-style: $style;
+    src: url('#{$url}#{$file}.eot');
+    src: url('#{$url}#{$file}.eot?#iefix') format('embedded-opentype'), url('#{$url}#{$file}.woff') format('woff'), url('#{$url}#{$file}.ttf')  format('truetype'), url('#{$url}#{$file}.svg#svgFontName') format('svg');
```
Comment (bgantick) [inline src/sass/tools/_fonts.scss]: This is assuming you have all formats. Some font sources will only give you `woff` and `woff2`

Comment (austinstarin) [reply src/sass/tools/_fonts.scss]: Should we pass in an array of formats in each function call?

Comment (abea) [reply src/sass/tools/_fonts.scss]: [Someone's take on passing through file types](https://gist.github.com/jonathantneal/d0460e5c2d5d7f9bc5e6). Might be helpful as a model.

Comment (abea) [reply src/sass/tools/_fonts.scss]: I submitted a PR to this branch adding that. https://github.com/punkave/frontend-starter/pull/33

**Comment by abea** [comment]
PR into this for file types: https://github.com/punkave/frontend-starter/pull/33

**Comment by abea** [comment]
Reopening with added parameter.

**Pull request merged**